### ISSUE TITLE
Options from environment variables

### ISSUE THREAD

**Issue by pthelusma** [Xabaril/AspNetCore.Diagnostics.HealthChecks#265]
Hello, using the docker image but I cannot seem to find a way to pass in environment variables or use appsettings to override the default UIPath and APIPath. Is there a way to do this already? Would this be a viable option to contribute to the codebase?

**Comment by peterbomers** [comment]
I have almost the same question
We like to do some branding of the docker image.
Is it possible to specify an inline css environment variable?
Or add a specific key to the AAC service?

**Comment by CarlosLanderas** [comment]
I have already thought about the CSS stuff. I'm gonna add some parameters for docker.

 For the API paths, let me take a look

**Comment by cilerler** [comment]
👍 for CSS environment variable

**Comment by CarlosLanderas** [comment]
This afternoon I'm gonna add this functionality.

The image will allow to configure custom css for branding using a volume and all the paths:


```
docker run

-v /c/temp/css:/app/css

-e ui_stylesheet=/app/css/dotnet.css

-e ui_path=/healthchecks-e ui_resources_path=/static

-e ui_api_path=/health-api 

-p 5000:80 xabarilcoding/healthchecksui:latest

```

**Comment by pthelusma** [comment]
Will the ui_path accept just "/" as well?

**Comment by CarlosLanderas** [comment]
@pthelusma, asp.net core middleware does not allow to bind "/" as a valid path. It throws an exception

**Comment by pthelusma** [comment]
Makes sense. Thank you

**Comment by CarlosLanderas** [comment]
Available in UI Version 3.0.3

PR: 

https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/332

Updated docs: 

https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/blob/master/doc/ui-docker.md
